language_server/src/dap/feature_launch.cpp

```diff
@@ -51,7 +51,7 @@ json feature_launch::register_capabilities()
 {
 	return json
 	{
-		{"supportsConfigurationDoneRequest", true}
+		{"supportsConfigurationDoneRequest", false}
```

Comment:
are you sure that is the only capabilities that you support?
https://microsoft.github.io/debug-adapter-protocol/specification#Types_Capabilities